torq.q

```diff
@@ -611,16 +611,29 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+// function to execute functions in .proc.initlist
 .proc.try:{[id;a]
-  .lg.o[id;"attemping to run: ",.Q.s1 a];
-  @[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a]}[id;a]];
+  .lg.o[id;"attempting to run: ",.Q.s1 a];
+  success:@[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a];:1b}[id;a]];
```

Comment:
I think success should be a Bool like so:
```
@[{value x;1b};a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a];0b}[id;a]];
```
You'll get `1b` if it runs and `0b` if it fails, which would make the next line:
```
if[not success;:()];
```